cla_public/templates/accessibility-statement.html

```diff
@@ -90,28 +103,50 @@ <h2 class="govuk-heading-m">{{ _('Content that’s not within the scope of the a
     {% trans %}Not applicable{% endtrans %}
   </p>
 
-  <h2 class="govuk-heading-m">{{ _('How we tested this service') }}</h2>
+  <h2 class="govuk-heading-m">{{ _('What we’re doing to improve accessibility') }}</h2>
 
   <p class="govuk-body">
-    {% trans %}This service was last tested on 22 July 2020. The test was carried out by the Digital Accessibility Centre (DAC).{% endtrans %}
+    {% trans %}We are working to fix the time-out issue listed in this statement.{% endtrans %}
   </p>
 
   <p class="govuk-body">
-    {% trans %}The DAC tested the service available at <a class="govuk-link" href="https://checklegalaid.service.gov.uk/">https://checklegalaid.service.gov.uk/</a>.{% endtrans %}
+    {% trans %}We will monitor the accessibility of this service on an ongoing basis and fix any accessibility issues reported to us.{% endtrans %}
   </p>
 
-  <h2 class="govuk-heading-m">{{ _('What we’re doing to improve accessibility') }}</h2>
+  <h2 class="govuk-heading-m">{{ _('Preparation of this accessibility statement') }}</h2>
 
   <p class="govuk-body">
-    {% trans %}We are working to fix the issues listed in this statement by December 2020. When the issues are fixed we will send the service for a re-test with the DAC.{% endtrans %}
+    {% trans
+      day = "17",
+      month = _('September'),
+      year = "2020"
+    %}This statement was prepared on {{ day }} {{ month }} {{ year }}.{% endtrans %}
+    {% trans
+      day = "30",
+      month = _('November'),
```

Comment:
I don't see any Welsh translation for `November`

Reply:
I'll put a placeholder in...